config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-release-0-4.yaml

```diff
@@ -0,0 +1,172 @@
+    testgrid-tab-name: capz-periodic-conformance-k8s-main-release-0-4
+    testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
+    description: Runs conformance & node conformance tests on latest kubernetes master with cluster-api-provider-azure
+- name: periodic-cluster-api-provider-azure-capi-e2e-release-0-4
+  decorate: true
+  decoration_config:
+    timeout: 4h
+  interval: 12h
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+    preset-azure-cred-only: "true"
+    preset-azure-anonymous-pull: "true"
+  extra_refs:
+    - org: kubernetes-sigs
+      repo: cluster-api-provider-azure
+      base_ref: release-0-4
+      path_alias: "sigs.k8s.io/cluster-api-provider-azure"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210201-77bfce8-1.18
+      command:
+      - "runner.sh"
+      - "./scripts/ci-e2e.sh"
+      env:
+        - name: GINKGO_FOCUS
+          value: "Cluster API E2E tests"
+        - name: LOCAL_ONLY
+          value: "false"
+      # docker-in-docker needs privileged mode
+      securityContext:
+        privileged: true
+  annotations:
+    testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
+    testgrid-tab-name: capz-periodic-capi-e2e-release-0-4
+    testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
+- name: periodic-cluster-api-provider-azure-coverage-release-0-4
+  interval: 12h
+  decorate: true
+  path_alias: "sigs.k8s.io/cluster-api-provider-azure"
+  extra_refs:
+    - org: kubernetes-sigs
+      repo: cluster-api-provider-azure
+      base_ref: release-0-4
+      path_alias: "sigs.k8s.io/cluster-api-provider-azure"
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210201-77bfce8-1.18
+      command:
+      - runner.sh
+      - bash
+      args:
+      - -c
+      - |
+        result=0
+        ./scripts/ci-test-coverage.sh || result=$?
+        cp coverage.* ${ARTIFACTS}
+        cd ../../k8s.io/test-infra/gopherage
+        GO111MODULE=on go build .
+        ./gopherage filter --exclude-path="zz_generated,generated\.go" "${ARTIFACTS}/coverage.out" > "${ARTIFACTS}/filtered.cov" || result=$?
+        ./gopherage html "${ARTIFACTS}/filtered.cov" > "${ARTIFACTS}/coverage.html" || result=$?
+        ./gopherage junit --threshold 0 "${ARTIFACTS}/filtered.cov" > "${ARTIFACTS}/junit_coverage.xml" || result=$?
+        exit $result
+      securityContext:
+        privileged: true
+  annotations:
+    testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
+    testgrid-tab-name: capz-periodic-coverage-release-0-4
+- name: periodic-cluster-api-provider-azure-e2e-full-release-0-4
+  decorate: true
+  decoration_config:
+    timeout: 2h
+  interval: 12h
```

Comment:
let's make the interval bigger here since we won't be merging stuff there that often. I think 24h should be good?

Reply:
changed all of the intervals in this file to 24h